src/interaction.js

```diff
@@ -144,34 +118,42 @@ export const extractParameters = (ixType) => {
  * @param {[string]} [props.signers] - list of signers, who will authorize transaction, specified as array of addresses.
  * @returns {Promise<any>}
  */
+
 export const sendTransaction = async (...props) => {
-  const extractor = extractParameters("tx");
-  const { code, args, signers, limit } = await extractor(props);
-  const serviceAuth = authorization();
-
-  // set repeating transaction code
-  const ix = [
-    fcl.transaction(code),
-    fcl.payer(serviceAuth),
-    fcl.proposer(serviceAuth),
-    fcl.limit(limit),
-  ];
-
-  // use signers if specified
-  if (signers) {
-    const auths = signers.map((address) => authorization(address));
-    ix.push(fcl.authorizations(auths));
-  } else {
-    // and only service account if no signers
-    ix.push(fcl.authorizations([serviceAuth]));
-  }
+  try {
+    const extractor = extractParameters("tx");
+    const { code, args, signers } = await extractor(props);
```

Comment:
`limit` was lost - probably during merge conflicts resolvement :)
```suggestion
    const { code, args, signers, limit } = await extractor(props);
```